bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -195,19 +227,21 @@ public void onValueReceived(@Nullable String variable, @Nullable String value, @
 
         updateStatus(ThingStatus.ONLINE);
 
+        String oldBinaryState = this.stateMap.get("BinaryState");
         if (variable != null && value != null) {
             this.stateMap.put(variable, value);
         }
 
-        if (getThing().getThingTypeUID().getId().equals("insight")) {
+        if ("insight".equals(getThing().getThingTypeUID().getId())) {
             String insightParams = stateMap.get("InsightParams");
 
             if (insightParams != null) {
                 String[] splitInsightParams = insightParams.split("\\|");
 
                 if (splitInsightParams[0] != null) {
-                    OnOffType binaryState = null;
-                    binaryState = splitInsightParams[0].equals("0") ? OnOffType.OFF : OnOffType.ON;
+                    logger.trace("New raw InsightParam binaryState '{}' for device '{}' received",
+                            splitInsightParams[0], getThing().getUID());
```

Comment:
I tried something similar before finding the real issue (see other comment). So just to not forget: Please remove this logging again as it probably provides no value.

Reply:
That was an easy one, changed line 235 to
        if ("insight1".equals(service))

Reply:
Still I would suggest this logging to be removed again as it doesn't provide any real value?

Reply:
Sorry, thought I had removed this. Done now